cdap-app-fabric/src/test/java/io/cdap/cdap/internal/app/services/http/handlers/SourceControlManagementHttpHandlerTests.java

```diff
@@ -94,7 +94,7 @@ public class SourceControlManagementHttpHandlerTests extends AppFabricTestBase {
   private static final String PASSWORD_NAME = "test_password_name";
   private static final String USERNAME = "test_user";
   private static final String TEST_FIELD = "test";
-  private static final String REPO_FIELD = "repository";
+  private static final String REPO_FIELD = "config";
```

Comment:
This was a bug in the failure tests. The tests were getting failed because we were generation wrong json rather than the config itself.